### PR TITLE
Don't store user objects or ticket objects

### PR DESCRIPTION
**Pull request by jarrydlee**

### CODE REVIEW

File: server/services/queue.js [hunk 1]
```diff
@@ -3,38 +3,36 @@ import queueManager from '../managers/queue';
...
+function createDelayedPage(ticketId, userId, device, delay, title) {
   const jobDetails = {
-    ticket,
-    user,
-    device: user.devices[deviceIndex],
-    title: ticket.metadata.title
+    ticketId,
+    userId,
+    device,
+    title: `${title} - ${device.name}`
```
Comment (kalebingerick) [inline server/services/queue.js:29]: Is this change to make the title better match the output Justin gave us?

File: server/tests/services/queue.js [hunk 2]
```diff
@@ -2,20 +2,28 @@ import queueService from '../../services/queue';
...
+        constants.user.id,
+        constants.user.devices[0],
+        1 * ONE_MINUTE_MILLISECOND,
+        constants.ticket.metadata.title).then(() => {
           expect(queueManager.testMode.jobs.length).to.equal(1);
           expect(queueManager.testMode.jobs[0].type).to.equal('page');
-          expect(queueManager.testMode.jobs[0].data.title).to.equal(constants.ticket.metadata.title);
-          expect(queueManager.testMode.jobs[0]._delay).to.equal(constants.user.delays[0] * ONE_MINUTE_MILLI);
+          expect(queueManager.testMode.jobs[0].data.title)
+            .to.equal(title + ' - ' + constants.user.devices[0].name);
```
Comment (kalebingerick) [inline server/tests/services/queue.js:24]: If this logic gets repeated in one more place, I'd like to see it removed into some kind of helper function, to DRY up the code.

File: server/services/queue.js (hunk 1 above)
Comment (jarrydlee) [reply server/services/queue.js:29]: It's so that you don't have multiple requests with the same title, just usability if they use the kue front end

**Comment by kalebingerick** [comment]
Good if the failing tests aren't an indication of actual failure

**Review by kalebingerick — APPROVED** [review]

**Pull request merged**